docs/contents.md

```diff
@@ -14,134 +14,143 @@ The **FortiAI** solution pack contains the following resources:
-| fortiai-static-questions | Contains questions for different modules along with field of interest to be used. |
+| Name                       | Description                                                                           |
+|:---------------------------|:--------------------------------------------------------------------------------------|
+| `fortiai-configurations`   | Contains keys like `llmIntegrations`, `llmIntegrationToUse`, and `llmIntegrationData` |
+| `fortiai-static-questions` | Contains questions for different modules along with field of interest to be used.     |
 
-### Table of Contents
+### Key Store Contents
 
 <table>
     <tr>  
-        <th rowspan=3>fortiai-configurations</th>  
-        <th>llmIntegrations</th>
+        <th rowspan=3><code>fortiai-configurations</code></th>
+        <th><code>llmIntegrations</code></th>
         <td>Sets the integration such as OpenAI, Google Bard, etc., to be used as the LLM. By default, 'OpenAI' is configured to be used as the LLM.
             <table>
                 <tr>
-                    <th>name</th>
+                    <th><code>name</code></th>
                     <td>LLM Integration name as per Connector. By default, `openai` is set.</td>
                 </tr>
                 <tr>
-                    <th>title</th>
+                    <th><code>title</code></th>
                     <td>LLM Integration title as per Connector. By default, `OpenAI` is set.</td>
                 </tr>
                 <tr>
-                    <th>modelList</th>
+                    <th><code>modelList</code></th>
                     <td>List of LLM Integration models. By default, `gpt-3.5-turbo`, `gpt-3.5-turbo-0301`, `gpt-4`, `gpt-4-1106-preview` is present.</td>
                 </tr>
             </table>
         </td>
     </tr>
     <tr>
-        <th>llmIntegrationToUse</th>
+        <th><code>llmIntegrationToUse</code></th>
         <td>Sets the integration such as OpenAI, Google Bard, etc., to be used as the LLM. By default, `OpenAI` is configured to be used as the LLM.</td>
     </tr>
     <tr> 
-        <th>llmIntegrationData</th>
+        <th><code>llmIntegrationData</code></th>
         <td>Contain the data for each LLM Integration.
             <table>
                 <tr>
-                    <th>conversationModel</th>
+                    <th><code>conversationModel</code></th>
                     <td>Sets the model of LLM Integration to use in <i>Conversation</i> and <i>Response Plans / Jinja / Playbook How To's </i> Mode. By Default, `gpt-4-1106-preview` is to be used. </td>
                 </tr>
                 <tr>
-                    <th>pBGenerationModel</th>
+                    <th><code>pBGenerationModel</code></th>
                     <td>Sets the model of LLM Integration to use in <i>Playbook Generation</i> Mode. By Default, `gpt-4-1106-preview` is to be used. </td>
```

Comment:
By Default, `gpt-4-1106-preview` --> needs to be replaed with latest

Reply:
Updated